components/integration/ballerina-vs-apollo-for-graphql/code/apollo-graphql-bbe/similarities-default-values-ballerina.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Default values"
-description: "Ballerina has in-built default value support, similar to GraphQL."
+description: "Ballerina has in-built default value support similar to GraphQL."
```

Comment:
The comma helps to separate the clauses and adds clarity to the sentence, making it easier to understand that the default value support in Ballerina is similar to that in GraphQL. @ThisaruGuruge wdyt?

Reply:
Yes, that was the idea. If it isn't grammatically wrong, let's keep it as it is.

Reply:
Reverted to the original.